exporter/redis.go

```diff
@@ -408,6 +408,8 @@ func (e *Exporter) scrape(scrapes chan<- scrapeResult) {
 		var c redis.Conn
 		var err error
 
+		scrapes <- scrapeResult{Name: "up", Addr: addr, Value: 0}
```

Comment:
You should be sending a given timeseries only once. Without testing it, redis_up will always be 0 and every time it's up Prometheus will also log a warning.